lib/persisted-model.js

```diff
@@ -825,8 +832,8 @@ module.exports = function(registry) {
 
       setRemoting(PersistedModel, 'updateLastChange', {
         description: [
-          'Update the properties of the most recent change record',
-          'kept for this instance.',
+          g.f('Update the properties of the most recent change record'),
+          g.f('kept for this instance.'),
         ],
```

Comment:
This is tricky.  Are the two strings concatenated?  Probably, so because each looks like a fragment of a sentence.  Then, it's the string concatenation globalization bug: make it one sentence string.